website/content/docs/getting-started/create.md

```diff
@@ -1,31 +1,37 @@
 ---
-title: "Create a Kind cluster"
+title: "Create a k0s cluster"
 draft: false
 weight: 1
 ---
 
+This section covers the steps needed to created a basic k0s cluster.
+
 #### Prerequisites
 
-Along with `boundless` CLI, you will also need the following tools:
+Along with `boundless` CLI, you will also need the following tools installed:
+
+* [k0sctl](https://github.com/k0sproject/k0sctl#installation) - required for installing a k0s distribution
+
+#### Download the example blueprint
+
+Download the [example blueprint](https://github.com/mirantiscontainers/boundless/tree/main/blueprints/k0s-example/k0s-example.yaml) to your machine.
+
+This blueprint is setup to install the cluster on your local machine without the need for any additional configuration.
+
+#### Apply the blueprint
 
-* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) - required for installing a kind distribution
+Applying the blueprint will create the cluster and install the components defined in the blueprint:
```

Comment:
create the cluster -> create the k0s cluster

install the components defined in the blueprint: -> install the components defined in the blueprint.